easybuild/easyconfigs/m/mdocfile/mdocfile-0.1.3-foss-2023a.eb

```
@@ -0,0 +1,33 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/10
+easyblock = 'PythonBundle'
+
+name = 'mdocfile'
+version = '0.1.3'
+
+homepage = 'https://github.com/teamtomo/mdocfile'
+description = """SerialEM mdoc files as pandas dataframes."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('pydantic','2.5.3')
+]
+
+use_pip = True
+
+exts_list = [
+    (name, version, {
+        'checksums': ['edb89f493ccdb6c04824e6c0cdeffe013bd20d95f03647ecb96f8740fd22bfc9'],
+    }),
+]
+
+sanity_pip_check = True
+
+moduleclass = 'bio'
+
```

remove trailing blank line
```suggestion
moduleclass = 'bio'
```